oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/requests/factory/v7/V78RequestFactory.java

```diff
@@ -45,7 +45,7 @@ public V78RequestFactory(final ElasticSearchVersion version) {
         template = new V78TemplateFactory(version);
         index = new V7IndexFactory(version);
         alias = new CommonAliasFactory(version);
-        document = new V7DocumentFactory(version);
+        document = new V8DocumentFactory(version);
```

Comment:
Is this a correct change?

Reply:
I think @zhenxu mean we should identify v8 ElasticSearch, and use a new v8 factory.